packages/bits/src/lib/time-picker/time-picker.component.ts

```diff
@@ -290,6 +290,10 @@ export class TimePickerComponent implements OnInit, OnDestroy, OnChanges, AfterV
     }
 
     ngOnDestroy() {
+        if (this.overlay.showing) {
+            this.overlay.hide();
+        }
```

Comment:
another instance where it would be good to add a unit test or some form of automation test

Reply:
Will added, thanks